packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,32 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/**
+ * FrameContexts provides information about the context in which the app is running within the host.
+ * Developers can use FrameContexts to determine how their app should behave in different contexts,
+ * and can use the information provided by the context to adapt the app to the user's needs.
+ * 
+ * @example 
+ * If the app is running in the settings context, the developer may want to display options for the 
+ * user to configure their preferences. If the app is running in the content context, the developer 
+ * may want to display information relevant to the content the user is currently viewing. 
+ */
 export enum FrameContexts {
+  /** App's frame context from where settings page can be accessed. */
   settings = 'settings',
+  /** The default context for the app where all the content of the app is displayed. */
   content = 'content',
+  /** Sets app's frame context to authentication flow for the app. */
```

Comment:
>Sets app's frame context to authentication flow for the app. [](http://example.com/codeflow?start=6&length=60)

Given how confusing authentication can be, providing a clearer example might be helpful, e.g.,

```
Frame context used when app is running in the authentication window launched by calling {@link authentication.authenticate}
```

Reply:
Will update in latest commit.